microsite/blog/2020-09-30-plugin-marketplace.md

```diff
@@ -0,0 +1,45 @@
+
+Now you may be thinking, “Yeah, sure, that sounds nice and all, but how does Backstage actually abstract away infrastructure?” The short answer: [plugins](https://backstage.io/plugins).
+
+![plugins](https://backstage.io/img/cards-plugins.png)
+
+<!--truncate-->
+
+Think of plugins as a mini version of an infrastructure tool or service — just the parts you need, made quick and easy. The advantage of using a Backstage plugin instead of a tool’s dedicated UI is that all your infrastructure is packaged into a “single pane of glass” instead of being spread out like independently built “islands”. Once you grow your infrastructure portfolio, the complexity really starts to add up. The Backstage plugin model allows you to add more tools without increasing the cognitive load for your users.
+
+Our goal for the plugin ecosystem is that anything can be a plugin. The definition of a plugin is flexible enough to allow you to expose pretty much any kind of infrastructure or software development tool as a plugin in Backstage. By following strong [design guidelines](https://backstage.io/blog/2020/09/30/backstage-design-system) we ensure the overall user experience stays consistent between plugins. If we all do our job well, the end user of Backstage does not see the boundaries between plugins. They are interacting with one consistent product — with many features.
+
+## Building an ecosystem
+
+Imagine a not-so-distant future where you walk up to Backstage, install it in your environment, and then choose from a gallery of existing, open source plugins that match whatever infrastructure you use inside your company. That is our vision for the Backstage plugin ecosystem.
+
+Running services on Kubernetes? There’s a plugin for that. Using Snyk for security scanning? There’s a plugin for that. Grafana, DataDog, or Rollbar for monitoring? There’s a plugin for that. Using Jenkins, CircleCI, GitHub Actions, or Travis CI? Pick the CI plugin of your choice. You get the picture. Regardless of your stack, there’s a plugin that you can use.
+
+Like this vision? So do we! But we can’t do it alone. We’ll need your help. For this vision to come true, we need to foster a community where many companies and individual developers build and contribute their plugins. Being [accepted into the CNCF Sandbox](https://backstage.io/blog/2020/09/23/backstage-cncf-sandbox) will provide us with the groundwork and guidance to broaden the community even faster.
+
+This grand vision is actually not that far off. Already today there is a growing ecosystem of Backstage plugins. To highlight all the amazing work that has been done by the community, and make it easier for Backstage adopters to see what plugins are available, we now have a Plugin marketplace:
+
+![marketplace](assets/marketplace.png)
+
+https://backstage.io/plugins
```

Comment:
Did you intend this to be a bare link all by itself? If we want to show the URL to the reader (instead of hiding it in a `[]()` link), should it be done in a sentence like for example `The plugin marketplace can be found at https://backstage.io/plugins`, or in the paragraph above the image, `...we now have a Plugin marketplace at https://backstage.io/plugins`?

Reply:
Fixed.